src/config/dynamic-theme-fixes.config

```diff
@@ -848,10 +848,38 @@ body, .body, table, tbody, tr, td, .corner_bottom {
 
 atlassian.net
 
+INVERT
+img[src*="https://github.githubassets.com/favicon.ico"].smart-link-icon
+.sc-jTzLTM /* Pre-loading comment avatar */
+
 CSS
 #jira-issue-header, #jira-issue-header-actions {
     background-color: transparent !important;
 }
+span.code /* Inline code span */ {
+    background-color: rgba(240,246,252,0.15) !important; 
+}
+.ghx-parent-group, /* Kanban board parent issue */
+.ghx-issue /* Kanban board issue*/
+{
+/* Important needed for .ghx-issue*/
+    background-color: rgba(119, 183, 255, 0.05) !important; 
+}
+span.loader-wrapper a, /* Smart links */
+.css-1ruu48h, /* Normal issue action button below title */
+.css-tmtnt9, /* Button attached to arrow down btn */
+.css-14v1uf5, /* Arrow down btn */ 
+.css-wvfva4 /* Three dots btn */
+{
```

Comment:
Move the `{` on the same line after `css-wvfva4`

Reply:
Done